components/permalink_view/permalink_view.test.jsx

```diff
@@ -178,6 +178,7 @@ describe('components/PermalinkView', () => {
             });
 
             test('should redirect to DM link with postId for permalink', async () => {
+                Date.now = () => new Date(0).getMilliseconds();
```

Comment:
Minor nit, but this will affect other test cases in this file as well. I don't think it'll cause any problems, but it's something to be aware of. Perhaps we should move it up to the start of the file so it's more obvious that it affects multiple cases instead of just affecting the tests after this one.

Reply:
Would `import {fakeDate} from 'tests/helpers/date';` work?

```ts
    let resetFakeDate: () => void;

    beforeEach(() => {
        resetFakeDate = fakeDate(new Date(...));
    });

    afterEach(() => {
        resetFakeDate();
    });
```

Reply:
(Or keep and restore the original reference before/after)